moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -148,7 +148,22 @@ const TABS = [
   },
 ];
 
-const history = createBrowserHistory();
+const DEFAULT_SNACKBAR_STATE = {
+  open: false,
+  message: "Default State",
+  severity: "warning",
+};
+
+/**
+ * Get the index of the currently active tab
+ * @param {*} tabName - a `tab` name from the url search string
+ * @returns {integer} - the TAB index of the currently active tab, falling back to `0`
+ */
+const useActiveTabIndex = (tabName) =>
+  useMemo(() => {
+    const activeTabIndex = TABS.findIndex((tab) => tab.param === tabName);
+    return activeTabIndex > -1 ? activeTabIndex : 0;
```

Comment:
if an unknown tab name is used in the search params, we fallback to the summary tab. this behavior was perviously broken. E.g.: https://moped.austinmobility.io/moped/projects/1921?tab=pizza

Reply:
its not falling back to the summary tab for me.

Reply:
oh duhh b/c i clicked your link and not the netlify link. ignore me.